src/validate-containers.js

```diff
@@ -63,19 +62,25 @@ const validateContainers = function (manifest) {
       // Check if environment variable with encoding is defined within private manifest
       const encoding = publicVars[envValue]['encoding']
       const privateVarSpec = privateManifest['vars'] && privateManifest['vars'][envValue]
+      if (!encoding) {
+        return
+      }
       if (encoding === 'private:sha256') {
-        if (!privateVarSpec) {
+        if (privateVarSpec) {
+        } else {
```

Comment:
Rather than having an if with an empty body and an else, you can use a single if that checks `!privateVarSpec`

Reply:
Right, meant to revert this change when taking out the interpolation code